wazo_chatd/database/models.py

```diff
@@ -45,6 +45,7 @@ class User(Base):
         nullable=False,
     )
     status = Column(Text())
+    do_not_disturb = Column(Boolean(), nullable=False, default=False)
```

Comment:
Your model is not the same as the migration script
```suggestion
    do_not_disturb = Column(Boolean(), nullable=False, server_default='false')
```